seeds/playlists.js

```diff
@@ -0,0 +1,9 @@
+exports.seed = function(knex, Promise) {
```

Comment:
I'm pretty unsure what the keyword Promise is doing in this line

Reply:
I'm pretty sure it's somehow defining what's expected to be returned?

Reply:
This article seems to suggest it isn't needed, which I would be inclined to believe:
https://stackoverflow.com/questions/52519636/is-the-promise-argument-passed-into-knex-migrations-needed